link/src/main/java/com/stripe/android/link/ui/signup/SignUpScreenState.kt

```diff
@@ -0,0 +1,18 @@
+package com.stripe.android.link.ui.signup
+
+import com.stripe.android.core.strings.ResolvableString
+import com.stripe.android.uicore.elements.PhoneNumberController
+import com.stripe.android.uicore.elements.TextFieldController
+
+sealed interface SignUpScreenState {
+    data class Content(
+        val emailController: TextFieldController,
+        val phoneNumberController: PhoneNumberController,
+        val nameController: TextFieldController,
```

Comment:
is it possible to not use these controllers? I feel like it should be possible to use their validation logic without actually using the controllers + UI directly. Motivation here is just that these are overly complicated imo, when we know exactly the fields we need to use and how they should look

Reply:
I tried to avoid the controllers, but the textfields associated with these are complicated, especially `PhoneNumberTextField`. Creating `TextField`s that match the existing would be increase the scope.